package.json

```diff
@@ -24,15 +24,22 @@
     {
       "name": "Tim Gerstel",
       "email": "tgerstel@rocketsoftware.com"
+    },
+    {
+      "name": "Leonty Chudinov",
+      "email": "lchudinov@rocketsoftware.com"
     }
   ],
   "main": "lib/index.js",
   "scripts": {
-    "build": "tsc"
+    "build": "tsc",
+    "start": "tsc --watch",
+    "test:storage": "mocha --timeout 10000 ./test/storage/*.js"
   },
   "dependencies": {
     "@rocketsoftware/express-ws": "^5.0.0",
     "accept-language-parser": "~1.5.0",
+    "axios": "^0.21.1",
```

Comment:
To prevent version accidental version updates that change behavior, we use `~`.

Reply:
I fixed the version string.